.travis.yml

```diff
@@ -1,20 +1,13 @@
 language: go
 
 go:
-  - '1.11.1'
+  - '1.11.13'
```

Comment:
Why did you change the Go 1.11.x version?

Reply:
I didn't commented it, but because of https://github.com/golang/go/issues/30446 
1.11.4 would probably have fixed it but well, latest version cannot hurt.

Reply:
Gotcha. I was taking this change to mean that the lowest version of Go now supported is 1.11.13.

What do you think of including '1.11.4' as the first build target and mentioning in the docs that's it the minimum supported version? Then adding '1.11.x' and '1.12.x' for the latest version of both of those?

Reply:
Ok, let's use > 1.11.4.
1.11 will be removed soon I guess since PBS supports only the last 2 Go versions (your PR is on its way for 1.13 #1058 :))

Reply:
Ah, that's a very good point.